src/components/ProjectCard/Section/ProjectSection.styles.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  margin-bottom: 1rem;
+  gap: 1rem;
+`;
+
+export const Header = styled.div`
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  gap: 0.25rem;
+`;
+
+export const Title = styled.h2`
+  font-size: 1.5rem;
+  font-weight: bold;
```

Comment:
```suggestion
  font-weight: bold;
  text-align: center;
```
When shrinking the screen, the title breaks a bit. I think this will solve it
![Captura de tela de 2025-02-26 11-59-11](https://github.com/user-attachments/assets/6c994afa-e5ab-4e8c-9a45-99bcadf315d4)